### PR TITLE
Restructuring for example_1

### PR DESCRIPTION
**Pull request by christophfroehlich**
following #231

### CODE REVIEW

File: example_1/hardware/include/ros2_control_demo_example_1/visibility_control.h [hunk 1]
```diff
@@ -0,0 +1,56 @@
+// Copyright 2017 Open Source Robotics Foundation, Inc.
```
Comment (destogl) [inline example_1/hardware/include/ros2_control_demo_example_1/visibility_control.h:1]:
```suggestion
// Copyright 2021 ros2_control Development Team
```

File: example_1/ros2_control_demo_example_1.xml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<library path="ros2_control_demo_example_1">
```
Comment (destogl) [inline example_1/ros2_control_demo_example_1.xml:1]: Also, please delete from the old place.

File: example_1/description/launch/view_robot.launch.py [hunk 3]
```diff
@@ -0,0 +1,99 @@
+# Copyright 2021 Stogl Robotics Consulting UG (haftungsbeschränkt)
```
Comment (destogl) [inline example_1/description/launch/view_robot.launch.py:1]: you can delete this file from the old place (if exists)

File: example_1/bringup/launch/test_forward_position_controller.launch.py [hunk 4]
```diff
@@ -0,0 +1,41 @@
+# Copyright 2021 Stogl Robotics Consulting UG (haftungsbeschränkt)
```
Comment (destogl) [inline example_1/bringup/launch/test_forward_position_controller.launch.py:1]: you can also delete this from the old place.

**Review by destogl — COMMENTED** [review]
Great! Thanks!

File: example_1/package.xml [hunk 5]
```diff
@@ -1,11 +1,12 @@
...
 <?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
 <package format="3">
-  <name>ros2_control_demo_hardware</name>
+  <name>ros2_control_demo_example_1</name>
   <version>0.0.0</version>
-  <description>Demo package of `ros2_control` Hardware. This package provides example on how to define hardware of your own robot for ROS2 control. The package is used with `ros2_control_demo_bringup` from package `rrbot_description` and `diffbot_description`.</description>
+  <description>Demo package of `ros2_control` hardware for RRbot.</description>
 
   <maintainer email="denis@stogl.de">Denis Štogl</maintainer>
+  <maintainer email="christoph.froehlich@ait.ac.at">Dr.techn. Christoph Froehlich</maintainer>
```
Comment (destogl) [inline example_1/package.xml:9]:
```suggestion
  <maintainer email="denis.stogl@stoglrobotics.de">Dr.-Ing. Denis Štogl</maintainer>
  <maintainer email="christoph.froehlich@ait.ac.at">Dr.techn. Christoph Froehlich</maintainer>
  
  <author email="denis.stogl@stoglrobotics.de">Dr.-Ing. Denis Štogl</author>
```

**Review by destogl — APPROVED** [review]
I am very happy! Thanks!

File: example_1/package.xml [hunk 6]
```diff
@@ -1,11 +1,14 @@
...
 <package format="3">
-  <name>ros2_control_demo_hardware</name>
+  <name>ros2_control_demo_example_1</name>
   <version>0.0.0</version>
-  <description>Demo package of `ros2_control` Hardware. This package provides example on how to define hardware of your own robot for ROS2 control. The package is used with `ros2_control_demo_bringup` from package `rrbot_description` and `diffbot_description`.</description>
+  <description>Demo package of `ros2_control` hardware for RRbot.</description>
 
-  <maintainer email="denis@stogl.de">Denis Štogl</maintainer>
+  <maintainer email="denis.stogl@stoglrobotics.de">Dr.-Ing. Denis Štogl</maintainer>
+  <maintainer email="christoph.froehlich@ait.ac.at">Dr.techn. Christoph Froehlich</maintainer>
```
Comment (bmagyar) [inline example_1/package.xml:9]: I know I'm cherry picking but could we please drop the academic decorations from ROS package maintainer names?

**Review by bmagyar — APPROVED** [review]

File: example_1/package.xml (hunk 6 above)
Comment (christophfroehlich) [reply example_1/package.xml:9]: I'm fine with that ;)

**Comment by christophfroehlich** [comment]
Do we need the camera link, hokuyo mesh etc in these example repositories? I added the missing mesh to example_1, should I do that for all the others rrbot-examples too?

**Comment by bmagyar** [comment]
@christophfroehlich updated the base branch, now we have a conflict! :D

**Comment by christophfroehlich** [comment]
> @christophfroehlich updated the base branch, now we have a conflict! :D

fixed that.

**Pull request merged**